bids/layout/writing.py

```diff
@@ -36,7 +35,7 @@ def replace_entities(entities, pattern):
     were denoted in the provided pattern.
     """
     entities = entities.copy()  # make a local copy, since dicts are mutable
-    ents = re.findall(r'\{(.*?)\}', pattern)
+    ents = re.findall(r'{(.*?)\}', pattern)
```

Comment:
It seems like the open and close brace should both be escaped or neither should be.

Reply:
Yes this should be left alone. I had PEP8 flag this for me before and changing it broke stuff.